### PR TITLE
Reword and other small improvements to the new test docs.

### PR DESCRIPTION
**Pull request by dragos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,21 +76,14 @@ Note: If you leave out `-Dconfig.file`, the default configuration file under `sr
...
-modes for the suite itself: the DCOS mode and the local mode (the latter means on your machine enabled by docker).
+### Add a test to an existing suite
 
-For local mode for example there is an sbt task named mit used to run the tests in local mode.
-Then this sbt task when run it creates a MesosIntegrationTestRunner instance
-which then calls different runners for different deploy modes for the spark
-itself (like cluster,client, etc). Then these runners run individual specs like
-ClientModeSpec which contain the actual test code (other specific scala test
-based specs) and which finally are submitted to a mesos cluster via a SparkJobRunner instance.
+Locate the right place to add your test. Note that currently there are two *root* suites: `ClientModeSpec` and `ClusterModeSpec`, each one mixing-in other traits with actual tests. The right place would be inside one of the traits that are mixed-in.
```
Comment (skonto) [inline README.md]: there is another one for multiple dispatchers...

**Comment by skonto** [comment]
LGTM other than specific note... maybe re-phrase there two basic root suites...

File: README.md (hunk 1 above)
Comment (dragos) [inline README.md]: I think the multi-dispatcher runner still ends up calling the `ClusterModeSpec`:

https://github.com/typesafehub/mesos-spark-integration-tests/blob/master/test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala#L101

Comment (skonto) [inline README.md]: ok

**Pull request merged**